src/resources/elements/primeDesignSystem/_dialogs.scss

```
@@ -0,0 +1,85 @@
+@import "./styles.scss";
+
+/**
+ * These are the default styles for ppopup-modal dialogs.
+ */
+.ppopup-dialog {
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  min-width: 160px;
+
+  &.pcard.gradient {
+    @include pcardBgAndGradient($backgroundColor: $Border01);
+  }
+
+  > .header,
+  > .body,
+  > .footer {
```

Nitpick

```suggestion
  > .dialogHeader,
  > .dialogBody,
  > .dialogFooter {
```

Just a bit worried, that generic names like this can cause potential issues in the future.
For example, just now in the review of the discussions PR, there was a comment, that also had a `.body` class.
I mean, with correct scoping, the problem I'm worried about should never appear, but yeah, maybe good to be cautious.

---

yeah I share that fear cuz I've encountered the problem before

---

done